tests/tests/operation_tests.cpp

```diff
@@ -7058,21 +7045,21 @@ BOOST_FIXTURE_TEST_SUITE(operation_tests, clean_database_fixture)
 
             auto alice_acc = db->get_account("alice");
             auto meta = db->get<account_metadata_object, by_account>("alice");
-            BOOST_REQUIRE(meta.account == "alice");
-            BOOST_REQUIRE(meta.json_metadata == json);
-            BOOST_REQUIRE(alice_acc.last_account_update == now);
+            BOOST_CHECK(meta.account == "alice");
```

Comment:
It's better to use BOOST_CHECK_EQUAL and similarity macros. It's print actual values when check fails.